### DISCUSSION TITLE
Facing issue while running the Make Build command

### DISCUSSION THREAD

**Discussion by GokulThanu** [ActivityWatch/activitywatch#1016] [Q&A]
Hi Team,

I am new to python, i have tried to install the activity watcher as mentioned in the documentation, While running the command "Make Build" in the command prompt it shows an error 

(venv) C:\Users\GokulThanudhas\activitywatch>make build
The system cannot find the path specified.
The system cannot find the path specified.
The system cannot find the path specified.
The system cannot find the path specified.
The system cannot find the path specified.
The system cannot find the path specified.
The system cannot find the path specified.
The system cannot find the path specified.
The system cannot find the path specified.
The system cannot find the path specified.
The system cannot find the path specified.
The system cannot find the path specified.
The system cannot find the path specified.
The system cannot find the path specified.
The system cannot find the path specified.
The system cannot find the path specified.
The system cannot find the path specified.
The system cannot find the path specified.
The system cannot find the path specified.
The system cannot find the path specified.
The system cannot find the path specified.
The system cannot find the path specified.
The system cannot find the path specified.
The system cannot find the path specified.
The system cannot find the path specified.
The system cannot find the path specified.
The system cannot find the path specified.
The system cannot find the path specified.
if [ -e "aw-core/.git" ]; then \
                echo "Submodules seem to already be initialized, continuing..."; \
        else \
                git submodule update --init --recursive; \
        fi
-e was unexpected at this time.
make: *** [build] Error 255

Please suggest how to resolve this

**Comment by ErikBjare** [comment]
Your question is identical to someone who posted on the forum just a few days ago: https://forum.activitywatch.net/t/building-from-source-running-into-issue-on-the-make-command-windows-10/2803?u=erikbjare

The solution is to use git bash instead of Powershell.
